README.md

```diff
@@ -1,24 +1,180 @@
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# 📗 Table of Contents
 
-Things you may want to cover:
+<details>
+  <summary>📗 Table of Contents</summary>
+  <ul>
+    <li><a href="#-car-rental-front-end-">📖 Book-an-Appointment </a></li>
+    <li><a href="#-built-with-">🛠 Built With</a></li>
+    <li><a href="#-getting-started-">💻 Getting Started</a></li>
+    <li><a href="#-authors-">👥 Authors </a></li>
+    <li><a href="#-future-features-">🔭 Future Features</a></li>
+    <li><a href="#-contributing-">🤝 Contributing</a></li>
+    <li><a href="#️-show-your-support-">⭐️ Show your support </a></li>
+    <li><a href="#-acknowledgments-">🙏 Acknowledgments </a></li>
+    <li><a href="#-license-">📝 License</a></li>
+  </ul>
+</details>
 
-* Ruby version
+<br>
 
-* System dependencies
+<!-- PROJECT DESCRIPTION -->
 
-* Configuration
+# 📖 Book-an-Appointment <a name="about-project"></a>
 
-* Database creation
+**Book-an-Appointment** is an application for cars. Create an account, and find and book your dream car today.
 
-* Database initialization
 
-* How to run the test suite
+## 🛠 Built With <a name="built-with"></a>
 
-* Services (job queues, cache servers, search engines, etc.)
+### Tech Stack <a name="tech-stack"></a>
 
-* Deployment instructions
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+    <li><a href="https://www.rubyonrails.org/">Rails</a></li>
+  </ul>
+</details>
 
-* ...
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- Handles logic for storing and retrieving data from the database
+- Handles logic for returning data to the frontend
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🪵 Documentation <a name="key-features"></a>
+
+- [Api docs]()
+
+## 🪵 Link to Frontend <a name="key-features"></a>
+
+- [Frontend]( https://github.com/calebchris000/Book-An-Appointment-Frontend)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🛹 Kanban Board <a name="key-features"></a>
+
+- Kanban board link: [link](https://github.com/calebchris000/Book-An-Appointment-Frontend/projects/1)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+```sh
+  git
+ bundle install
+ Your favorite text editor
+```
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  git clone   https://github.com/dheerajsachdeva/Book-An-Appointment-Backend.git
+```
+
+### Install
+
+Install this project with:
+
+```sh
+   cd Book-an-Appointment_back
+  bundle install
+  rails: db:create 
+  rails: db:migrate
+```
```

Comment:
- [ ] OPTIONAL: Please, consider adding instructions to your readme generate the secret key base locally, otherwise your users won't be able to create the database:

![image](https://github.com/dheerajsachdeva/Book-An-Appointment-Backend/assets/57421823/8cba09fb-0ff0-4404-b317-a6a7bb341288)

These are the instructions:

```
1. Remove config/master.key and config/credentials.yml.enc if they exist.
2. Run in the terminal: EDITOR=code rails credentials:edit 
3. Close the editor that opens.
4. This command will create a new master.key and credentials.yml.enc if they do not exist.
```